.github/workflows/deployToWordPressSVN.yml

```diff
@@ -0,0 +1,21 @@
+name: Deploy to WordPress SVN
+on:
+  push:
+    tags:
+    - "*"
```

Comment:
Update this to specifically run ONLY on Stable Releases (Not drafts and not tags).